### PR TITLE
Update links & Telehealth README.md

### PR DESCRIPTION
**Pull request by kazydek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Choose a sample application that demonstrates the use of chat components.
...
-| [`telehealth`](telehealth-example/) | <br><ul><li>Doctor to patient chat</li><li>Light & dark themes</li></ul></br> | 1:1 | <img src="/assets/telehealth-ios-app.png" alt="Telehealth sample application" style="width:300px"/> |
+| [`telehealth`](telehealth-example/) | <br><ul><li>Doctor-to-patient chat</li><li>Light & dark themes</li></ul></br> | 1:1 | <img src="/assets/telehealth-ios-app.png" alt="Telehealth sample application" style="width:300px"/> |
 
 
 ## Useful links
 
 For more information about PubNub Chat Components for iOS, refer to:
 * [GitHub repository](https://github.com/pubnub/chat-components-ios/blob/master/README.md)
-* [Documentation](https://www.pubnub.com/docs/chat/components/ios/get-started-ios)
+* [Documentation](https://www.pubnub.com/docs/chat/components/ios/get-started)
```
Comment (maciej-adamczyk) [inline README.md]: this is not the correct link to get-started doc

File: getting-started/README.md [hunk 2]
```diff
@@ -15,11 +15,11 @@ To use the app, you need:
 
 ## Usage
 
-Read the [Getting Started](https://www.pubnub.com/docs/chat/components/ios/get-started-ios) guide to learn how to use the app and better understand the logic behind it.
+Read the [Getting Started](https://www.pubnub.com/docs/chat/components/ios/get-started) guide to learn how to use the app and better understand the logic behind it.
```
Comment (maciej-adamczyk) [inline getting-started/README.md]: the same as above, previous one was correct

File: telehealth-example/README.md [hunk 3]
```diff
@@ -15,17 +15,19 @@ To use the app, you need:
 
 ## Usage
 
-Read the [Getting Started](https://www.pubnub.com/docs/chat/components/ios/get-started-ios) guide to learn how to use the app and better understand the logic behind it.
+Read the [Getting Started](https://www.pubnub.com/docs/chat/components/ios/get-started) guide to learn how to use the app and better understand the logic behind it.
```
Comment (maciej-adamczyk) [inline telehealth-example/README.md]: same here

**Review by maciej-adamczyk — APPROVED** [review]

File: telehealth-example/README.md (hunk 3 above)
Comment (kazydek) [reply telehealth-example/README.md]: Fixed.

File: getting-started/README.md (hunk 2 above)
Comment (kazydek) [reply getting-started/README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (kazydek) [reply README.md]: Fixed.

**Pull request merged**